.buildkite/configs/cleanup.gcp.yml

```diff
@@ -0,0 +1,40 @@
+---
+version: "1.0"
+
+accounts:
+  - name: "${ACCOUNT_PROJECT}"
+    driver: "gce"
+    options:
+      key: "${ACCOUNT_KEY}"
+      secret: "${ACCOUNT_SECRET}"
+      project: "${ACCOUNT_PROJECT}"
+
+scanners:
+  - account_name: "${ACCOUNT_PROJECT}"
+    resources:
+      - type: "node"
+        regions:
+          - "us-east1"
+        filters:
+          - type: "<"
+            pointer: "/extra/creationTimestamp"
+            param: "${CREATION_DATE}"
+            converters:
+              param: "date"
+              value: "date"
+          - type: "="
+            pointer: "/extra/labels/repo"
+            param: "elastic-package"
+          - type: "="
+            pointer: "/extra/labels/environment"
+            param: "ci"
+          - type: "regex"
+            pointer: "/name"
+            param: "^elastic-package-(.*)"
+          - type: "!="
+            pointer: "/state"
+            param: "unknown"
+          - type: "!="
+            pointer: "/state"
+            param: "terminated"
```

Comment:
Are states named the same in AWS and GCP?

Reply:
Found some examples, and those values were the same for AWS and GCP (terminated and unknown).

For instance in GCP: https://github.com/elastic/elastic-agent/blob/f9310c7f664bb25f37d671cca4e2fcf3749732c8/.buildkite/misc/gce-cleanup.yml#L36

Another example found for AWS:
```yaml
      - account_name: 'CLOUD_ACCOUNT_NAME'
        resources:
          - type: 'node'
            regions:
              - 'us-east-1'
            filters:
              - type: '<'
                pointer: '/created_at'
                param: '${DELETE_RESOURCE_BEFORE_DATE}'
                converters:
                  param: 'date'
              - type: 'regex'
                pointer: '/name'
                param: 'elastic-package-test-(.*)'
              - type: '!='
                pointer: '/state'
                param: 'unknown'
              - type: '!='
                pointer: '/state'
                param: 'terminated'
```